.rubocop.yml

```diff
@@ -0,0 +1,1157 @@
+AllCops:
```

Comment:
This is not a standard config. I had to change target Ruby version. Can i do that with `prepare`?